src/main/java/com/mongodb/shardsync/command/compare/CompareChunksCommand.java

```diff
@@ -0,0 +1,50 @@
+package com.mongodb.shardsync.command.compare;
+
+import com.mongodb.shardsync.ShardConfigSync;
+import com.mongodb.shardsync.SyncConfiguration;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.util.concurrent.Callable;
+
+@Command(name = "chunks", description = "Compare chunks between source and destination")
+public class CompareChunksCommand implements Callable<Integer> {
+    
+    @CommandLine.ParentCommand
+    private CompareCommand parent;
+    
+    @Option(names = {"--equivalent"}, 
+            description = "Check for chunk equivalence")
+    private boolean equivalent;
+    
+    @Option(names = {"--counts"}, 
+            description = "Compare chunk counts only")
```

Comment:
What happens if both this and `--equivalent` get specified?

Would it make more sense for this to be a string value rather than a boolean? `--compareMode=[full,countsOnly,none]`